packages/swapper/src/swappers/errors.ts

```diff
@@ -0,0 +1,6 @@
+import { SWAPPER_ERRORS } from './baseErrors'
+import { JUPITER_ERRORS } from './JupiterSwapper/errors'
+
+export { SWAPPER_ERRORS }
```

Comment:
see https://github.com/shapeshift/web/pull/8433/files#r1894285675 re: import/export in the same file, feels like something we may not want to do?

Reply:
Not relevant anymore, changed in [17f168a](https://github.com/shapeshift/web/pull/8433/commits/17f168a1ba52efd3e9b3f5e55e1b8f1e2266db37)